src/v4/mobx.ts

```diff
@@ -35,7 +35,7 @@ try {
     if (
         testCodeMinification.name !== "testCodeMinification" &&
         process.env.NODE_ENV !== "production" &&
-        process.env.IGNORE_MOBX_MINIFY_WARNING !== "true"
+        typeof process !== 'undefined' && process.env.IGNORE_MOBX_MINIFY_WARNING !== "true"
```

Comment:
let's put it as first or second condition, so that the other one can't died either (there have been reports of that in the past)

Reply:
Not needed, the first one is correctly replaced in latest version, see pictures in my comment.